src/DevLicenseCore.sol

```diff
@@ -201,6 +243,15 @@ contract DevLicenseCore is IDevLicenseDimo, AccessControl {
         return timestampCurrent - timestampInit <= _periodValidity;
     }
 
+    /**
+     * @notice It returns the alias associated with a token ID
+     * @dev It returns an empty string if no alias is associated with the token ID
+     * @param tokenId The unique identifier for the license token.
+     */
+    function getLicenseAlias(uint256 tokenId) public view returns (bytes32 licenseAlias) {
```

Comment:
Maybe a function for going the other way, too?

Reply:
Shame on me